magento/magento2
# 8255: Export Products action doesn't consider hide_for_product_page value.

### Description
Fix Export and Import Products action doesn't consider hide_for_product_page value.

### Fixed Issues (if relevant)

1. magento/magento2#8255: Export Products action doesn't consider hide_for_product_page value

### Manual testing scenarios
1. Create a simple product and upload an image.
2. Change the scope to 1 store view.
3. Set the image as Hide for product page in that store view.
4. Navigate to System -> Export.
5. Select Products and then Continue.
6. Check that CSV files should have 2 lines. The default values at the first row of values.
7. The second lines should contain values on sku, store_view_code and hide_from_product_page columns.
8. Remove created simple product.
9. Navigate to System -> Import.
10. Import product from exported CSV with add/update behavior.
11. Check, imported product at 1 store view has "Hide fro product page" checked for image.

### Contribution checklist
